### ISSUE TITLE
WooCommerce Orders Undefined array key "weight"

### ISSUE THREAD

**Issue by sjoukedv** [myparcelnl/woocommerce#741]
**Describe the bug**
WooCommerce order page throwing PHP errors

```
wp-content/plugins/woocommerce-myparcel/includes/admin/OrderSettings.php:471
OrderSettings->setLargeFormat()
wp-content/plugins/woocommerce-myparcel/includes/admin/OrderSettings.php:254
OrderSettings->setAllData()
wp-content/plugins/woocommerce-myparcel/includes/admin/OrderSettings.php:138
OrderSettings->__construct()
wp-content/plugins/woocommerce-myparcel/includes/admin/class-wcmypa-admin.php:405
WCMYPA_Admin->showMyParcelSettings()
wp-includes/class-wp-hook.php:303
do_action('woocommerce_admin_order_actions_end')
wp-content/plugins/woocommerce/includes/admin/list-tables/class-wc-admin-list-table-orders.php:312
WC_Admin_List_Table_Orders->render_wc_actions_column()
wp-content/plugins/woocommerce/includes/admin/list-tables/abstract-class-wc-admin-list-table.php:261
WC_Admin_List_Table->render_columns()
wp-includes/class-wp-hook.php:303
do_action('manage_shop_order_posts_custom_column')
wp-admin/includes/class-wp-posts-list-table.php:1343
WP_Posts_List_Table->column_default()
wp-admin/includes/class-wp-list-table.php:1452
WP_List_Table->single_row_columns()
wp-admin/includes/class-wp-posts-list-table.php:1377
WP_Posts_List_Table->single_row()
wp-admin/includes/class-wp-posts-list-table.php:795
WP_Posts_List_Table->_display_rows()
wp-admin/includes/class-wp-posts-list-table.php:772
WP_Posts_List_Table->display_rows()
wp-admin/includes/class-wp-list-table.php:1364
WP_List_Table->display_rows_or_placeholder()
wp-admin/includes/class-wp-list-table.php:1291
WP_List_Table->display()
wp-admin/edit.php:476
```

**Steps to reproduce this behaviour**
1. Visit WooCommerce > Orders (20 items/page)

**Expected behaviour**
Catch error or resolve setting if field is not set?

**Screenshots**
Setting:

![image](https://user-images.githubusercontent.com/24683344/145866547-a08b5596-6849-4716-9bbd-213736e40b6b.png)

**Details**
 - WordPress version: 5.8.2
 - WooCommerce version: 5.9.0
 - WooCommerce MyParcel version: 4.6.0
 - Theme: Divi
 - Hosting env: OpenLitespeed + PHP8

Please let me know if you would like me to add something!

**Comment by sjoukedv** [comment]
@Mitchell-MyParcel fixed in `>4.7.0`?

**Comment by EdieLemoine** [comment]
This was resolved in release 4.7.0 (#721)
